lib/barbeque/message/job_execution.rb

```diff
@@ -13,7 +13,11 @@ def assign_body(message_body)
         super
         @application = message_body['Application']
         @job  = message_body['Job']
-        @body = message_body['Message']
+        @body = if message_body['Message'].is_a?(String)
```

Comment:
Why are both `Barbeque::Message::JobExecution` and `Barbeque::Message::Notification` modified? Isn't changing only Notification sufficient?